code/modules/power/singularity/collector.dm

```diff
@@ -23,16 +24,27 @@
 	var/active = 0
 	var/locked = FALSE
 	var/drainratio = 0.5
-	var/powerproduction_drain = 0.001
-
+	var/powerproduction_drain = 0.01
 	var/bitcoinproduction_drain = 0.15
 	var/bitcoinmining = FALSE
+	var/obj/item/radio/radio
+	var/radio_key = /obj/item/encryptionkey/headset_eng
+	var/radio_channel = RADIO_CHANNEL_ENGINEERING
```

Comment:
hardcode those, they don't need to be variables

Reply:
I just copy-pasted the radio code from the cryo cells and every other console that uses radios does it like this too, I don't see why we'd need to snowflake it for this machine.

Reply:
I also don't know how to do this.